src/gosling-mouse-event/polygon.ts

```diff
@@ -1,3 +1,31 @@
+import { pointsToDegree } from '../core/utils/polar';
+
+/**
+ * @param {Array} point Tuple of the form `[x,y]` to be tested.
```

Comment:
It is usually preferred when using TS with JSDoc to omit the type definition (within `{ }`) in the JSDoc comment. e..,g

```
 * @param point Tuple of the form `[x,y]` to be tested.
 ```

This is because the TypeScript type is much more precise, and the JSDoc comment is ignored by the typechecker completely. We have some JSDoc comments like this in Viv, but that's because our documentation generator relies on the type in the JSDoc (and I'd prefer to get rid of it).